ext/js/language/translator.js

```diff
@@ -1609,7 +1609,7 @@ export class Translator {
             sequence: rawSequence,
             rules
         } = databaseEntry;
-        // cast is safe because getDeinflections filters out deinflection definitions
+        // Cast is safe because getDeinflections filters out deinflection definitions
```

Comment:
Not directly related to this PR, but I wonder if there's a lint that prevents casts, so one would be forced to leave a line of justification about why it's safe like this

Reply:
I've thought about that too, I can look around and see if anything in typescript-eslint helps with that when I get back to that.